src/views/catalog/utils/useWizardVmCreate.ts

```diff
@@ -52,6 +56,9 @@ export const useWizardVmCreate = (): UseWizardVmCreateValues => {
           devices.logSerialConsole = false;
           vmDraft.spec.template.spec.domain.devices = devices;
         }
+
+        if (!vmDraft.spec.template.metadata.labels) vmDraft.spec.template.metadata.labels = {};
```

Comment:
`getLabels` would be useful here as well.

Reply:
ok